### PR TITLE
chore(renaming): rename libs folder in packages

### PR DESCRIPTION
**Pull request by andypf**

### CODE REVIEW

**Review by edda — APPROVED** [review]

File: apps/template/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "template",
+  "name": "@cloudoperators/juno-app-template",
```
Comment (ArtieReus) [inline apps/template/package.json:2]: can we also update the package-lock file

File: packages/template-lib/package.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "template-lib",
+  "name": "@cloudoperators/juno-template-lib",
```
Comment (ArtieReus) [inline packages/template-lib/package.json:2]: please update also package-lock.json

File: packages/communicator/package.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "communicator",
+  "name": "@cloudoperators/juno-communicator",
```
Comment (ArtieReus) [inline packages/communicator/package.json:2]: please update also package-lock.json

**Review by ArtieReus — CHANGES_REQUESTED** [review]

**Review by ArtieReus — APPROVED** [review]
lgtm

**Pull request merged**